File: Backend/Models/MovieLanguage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class MovieLanguage
    {
        public string MovieLanguageId { get; set; }

        public string Language { get; set; }

        public ICollection<Movie> Movies { get; set; }
    }
}

File: Frontend/src/domain/Movie/MovieLanguage.ts
import { MovieLanguageDTO } from "../dto/MovieLanguageDTO";

export class MovieLanguage {
    public movieLanguageId: string;
    public language: string;
  
    constructor(data: MovieLanguageDTO) {
      this.movieLanguageId = data.movieLanguageId;
      this.language = data.language;
    }
  }